inference-engine/src/multi_device/multi_device_plugin.cpp

```diff
@@ -268,7 +266,7 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
                  }
              }
              iter->config = deviceConfig;
-             strDevices = iter->deviceName;
+             strDevices += iter->deviceName;
```

Comment:
@myshevts  fix an error here

Reply:
likely you need another PR

Reply:
@myshevts What does it mean "another PR"? If anything is missed for https://jira.devtools.intel.com/browse/CVS-55563?